include/aspect/material_model/rheology/visco_plastic.h

```diff
@@ -232,6 +270,14 @@ namespace aspect
 
 
         private:
+    
+      /**
```

Comment:
Can you specify what this is needed for in `visco_plastic`? Rather than specify them here, I think you can retrieve them with through simulator access or another method?